### ISSUE TITLE
integrate into joplin directly

### ISSUE THREAD

**Issue by Wladefant** [hieuthi/joplin-plugin-copy-anchor-link#1]
Why is that feature not integrated into joplin? @laurent22 are you opposed to that? I mean that is a feature that makes life easier and does not hurt anyone and I think many would use it. At a minimum it is a better workaround than copying the id, the going to note, then remembering the syntax and then writing, as it is right now. 

And a very important is, that it gets integrated into the PHONE APP.

Many note taking apps has this function with different approaches, but never with such difficult a approach as joplin. for example notion has a button "copy link to block" for example. But this approach presented by @hieuthi is the best i think.

@hieuthi made it like in github, am i correct? If so I think it needs to be polished before integrating. For example It would be better if the icon would be smaller, like github does 
![image](https://github.com/hieuthi/joplin-plugin-copy-anchor-link/assets/79611093/cb258db6-0826-4c67-83e5-7e95ab70d9d9). I would also suggest to darken the color. And of course the main thing is to remove the letters 
![image](https://github.com/hieuthi/joplin-plugin-copy-anchor-link/assets/79611093/3c43fc82-30e7-42c4-ab75-91e33c690085)

**Comment by ioogithub** [comment]
I agree with the comment bout the icon it is way to big and fuzzy, not clear and sharp.
